src/pages/ReportDetailsPage.tsx

```diff
@@ -198,6 +220,29 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         />
     ) : null;
 
+    const renderAvatar = isGroupChat ? (
+        <AvatarWithImagePicker
+            source={icons[0].source}
+            isUsingDefaultAvatar={!report.avatarUrl}
+            size={CONST.AVATAR_SIZE.XLARGE}
+            avatarStyle={styles.avatarXLarge}
+            disableViewPhoto
+            onImageRemoved={() => {
+                // Calling this without a file will remove the avatar
+                Report.updateGroupChatAvatar(report.reportID ?? '');
+            }}
+            onImageSelected={(file) => Report.updateGroupChatAvatar(report.reportID ?? '', file)}
+            editIcon={Expensicons.Camera}
+            editIconStyle={styles.smallEditIconAccount}
+        />
+    ) : (
+        <RoomHeaderAvatars
+            icons={icons}
+            reportID={report?.reportID}
+        />
+    );
+
+    const reportName = isGroupChat ? ReportUtils.getGroupChatName(undefined, true, report.reportID ?? '') : ReportUtils.getReportName(report);
```

Comment:
This potentially does a sort of all participants on every render, which seems like it might be slow for large groups, so I suggest memoizing it